### PR TITLE
[yarn test] add deployMode to examples

### PR DESCRIPTION
**Pull request by piaolaidelangman**

### CODE REVIEW

File: python/orca/example/learn/pytorch/cifar10/cifar10.py [hunk 1]
```diff
@@ -48,12 +48,14 @@
 
 if args.cluster_mode == "local":
     init_orca_context(memory="4g")
-elif args.cluster_mode == "yarn":
+elif args.cluster_mode == "yarn-client":
```
Comment (qiuxin2012) [inline python/orca/example/learn/pytorch/cifar10/cifar10.py:51]: if cluster_mode startwith yarn

File: python/orca/example/learn/pytorch/super_resolution/super_resolution.py [hunk 2]
```diff
@@ -64,10 +64,14 @@
 
 if opt.cluster_mode == "local":
     init_orca_context()
-elif opt.cluster_mode == "yarn":
+elif opt.cluster_mode == "yarn-client":
```
Comment (qiuxin2012) [inline python/orca/example/learn/pytorch/super_resolution/super_resolution.py:67]: same

File: python/orca/example/torchmodel/train/imagenet/main.py [hunk 3]
```diff
@@ -84,29 +84,23 @@ def main():
...
-            conf={"spark.rpc.message.maxSize": "1024",
-                  "spark.task.maxFailures": "1",
-                  "spark.driver.extraJavaOptions": "-Dbigdl.failure.retryTimes=1"})
+    
+    # init
+    hadoop_conf = os.environ.get("HADOOP_CONF_DIR")
+    assert hadoop_conf, "Directory path to hadoop conf not found for yarn-client mode. Please " \
+            "set the environment variable HADOOP_CONF_DIR"
+
+    conf = create_spark_conf().set("spark.executor.memory", args.executor_memory)\
```
Comment (qiuxin2012) [inline python/orca/example/torchmodel/train/imagenet/main.py:98]: use init_orca_context's build-in parameter

File: python/orca/example/torchmodel/train/resnet_finetune/resnet_finetune.py [hunk 4]
```diff
@@ -51,24 +52,24 @@ def forward(self, x):
...
+    parser.add_option("--executor-memory", type=str, dest="executorMemory", default="30g", help="executor memory")
+    parser.add_option("--driver-memory", type=str, dest="driverMemory", default="30g", help="driver memory")
+    parser.add_option("--deploy-mode", type=str, dest="deployMode", default="yarn-client", help="yarn deploy mode, yarn-client or yarn-cluster")
+    (options, args) = parser.parse_args(sys.argv)
+
+    hadoop_conf = os.environ.get('HADOOP_CONF_DIR')
+    assert hadoop_conf, "Directory path to hadoop conf not found for yarn-client mode. Please " \
+            "set the environment variable HADOOP_CONF_DIR"
+
+    conf = create_spark_conf().set("spark.executor.memory", options.executorMemory)\
```
Comment (qiuxin2012) [inline python/orca/example/torchmodel/train/resnet_finetune/resnet_finetune.py:67]: use init_orca_context's build-in parameter

Comment (piaolaidelangman) [reply python/orca/example/torchmodel/train/resnet_finetune/resnet_finetune.py:67]: OK, I will update.

**Comment by piaolaidelangman** [comment]
I hava updated.

**Pull request closed without merging**